templates/eap-cd-sso-s2i.json

```diff
@@ -477,7 +477,22 @@
                         "uri": "${SOURCE_REPOSITORY_URL}",
                         "ref": "${SOURCE_REPOSITORY_REF}"
                     },
-                    "contextDir": "${CONTEXT_DIR}"
+                    "contextDir": "${CONTEXT_DIR}",
+                    "images": [
+                        {
+                            "from": {
+                                "kind": "ImageStreamTag",
+                                "namespace": "${IMAGE_STREAM_NAMESPACE}",
+                                "name": "redhat-sso-cd-openshift:1.0"
+                            },
+                            "paths": [
+                                {
+                                    "destinationDir": "./${CONTEXT_DIR}/extensions/sso-adapters",
+                                    "sourcePath": "/opt/rh/rh-sso/client/eap7"
```

Comment:
I think the issue is that this needs to end with `/.` (slash dot, i.e. `/opt/rh/rh-sso/client/eap7/.`), which copies the contents of the directory into the target location.  See here: https://docs.openshift.com/container-platform/3.9/dev_guide/builds/build_inputs.html#image-source